master/getting-started/rkt/installation/manual.md

```diff
@@ -0,0 +1,173 @@
+
+- The Calico per-node rkt container, [calico/node](https://quay.io/repository/calico/node?tab=tags)
+- The [`calicoctl`](https://github.com/projectcalico/calico-containers) binary used to manage network
+  policy.
+- The [calico-cni](https://github.com/projectcalico/calico-cni) network plugin binaries.
+  - This is the combination of two binary executables and a configuration file.
+
+The `calico/node` rkt container must be run on node in your cluster.  It contains 
+the BGP agent necessary for Calico routing to occur, and the Felix agent which programs 
+network policy rules.
+
+The `calicoctl` binary is a command line utility that can be used to manage network
+policy for your rkt containers, and can be used to monitor the status of your
+Calico services.
+
+The `calico-cni` plugin binaries integrate directly with the rkt container lifecycle 
+hooks on each node to configure the container interfaces, manage IP addresses and 
+configure the containers to use Calico policy.
+
+## Installing `calico/node` 
+
+### Prepare host directory structure
+
+The `calicoctl` binary uses certain known directories for service diagnostics and
+status discovery.  In addition, this tutorial assumes binaries and CNI network
+configuration will be placed in a particular location.
+
+Ensure the following directories are created by running the following commands on
+each node:
+
+```
+mkdir -p /var/run/calico
+mkdir -p /var/log/calico
+mkdir -p /opt/bin
+mkdir -p /etc/rkt/net.d
+```
+
+### Run `calico/node` and configure the node.
+
+Each Calico-rkt enabled node requires the `calico/node` container to be running.
+
+The calico/node container can be run directly through rkt and needs to be run as
+as fly stage-1 container.
+
+```shell
+sudo rkt run --stage1-path=/usr/share/rkt/stage1-fly.aci \
+  --set-env ETCD_ENDPOINTS=http://<ETCD_IP>:<ETCD_PORT> \
+  --set-env IP=autodetect \
+  --insecure-options image \
+  --volume birdctl,kind=host,source=/var/run/calico,readOnly=false \
+  --mount volume=birdctl,target=/var/run/calico \
+  --volume mods,kind=host,source=/lib/modules,readOnly=false  \
+  --mount volume=mods,target=/lib/modules \
+  --volume logs,kind=host,source=/var/log/calico,readOnly=false \
+  --mount volume=logs,target=/var/log/calico \
+  --net host \
+  quay.io/calico/node:latest &
+```
+
+> Replace `<ETCD_IP>:<ETCD_PORT>` with your etcd configuration.  The `ETCD_ENDPOINTS`
+> environment may contain a comma separated list of endpoints of your etcd cluster.
+> If the environment is omitted, Calico defaults to a single etcd 
+> endpoint at http://127.0.0.1:2379.
+
+You can check that it's running using `sudo rkt list`.
+
+```shell
+$ sudo rkt list
+UUID      APP	IMAGE NAME                  STATE   CREATED         STARTED         NETWORKS
+b52bba11  node  quay.io/calico/node:latest  running 10 seconds ago  10 seconds ago
+```
+
+## Intalling the calicoctl CLI tool
+
+Download the calicoctl binary and ensure it is writeable.  We download to the 
```

Comment:
I'm complaining that it doesn't need to be writeable, it needs to be executable, right?